akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteNodeShutdownAndComesBackSpec.scala

```diff
@@ -118,8 +118,8 @@ abstract class RemoteNodeShutdownAndComesBackSpec
         watch(subjectNew)
 
         subjectNew ! "shutdown"
-        fishForMessage(5.seconds) {
-          case _: ActorIdentity       ⇒ false
+        receiveWhile(5.seconds) {
+          case _: ActorIdentity       ⇒ true
```

Comment:
Might be my mistake, but I understood receiveWhile such that it is the matching that matters, not the return value of the PF?